src/test/scala/is/hail/io/ExportVcfSuite.scala

```diff
@@ -95,6 +93,20 @@ class ExportVcfSuite extends SparkSuite {
     p.check()
   }
 
+  @Test def testEmptyReadWrite() {
+    var s = FilterVariantsAll.run(State(sc, sqlContext, LoadVCF(sc, "src/test/resources/sample.vcf")))
+    val out = tmpDir.createTempFile("foo", "vcf")
+    val out2 = tmpDir.createTempFile("foo", "vcf.bgz")
+
+    ExportVCF.run(s, "-o", out)
+    ExportVCF.run(s, "-o", out2)
+
+    assert(hadoopConf.getFileSize(out) > 0)
+    assert(hadoopConf.getFileSize(out2) > 0)
+    val rb1 = ImportVCF.run(s, out).vds
+    val rb2 = ImportVCF.run(s, out2).vds
+  }
+
   @Test def testPPs() {
     var s = State(sc, sqlContext)
     s = ImportVCF.run(s, Array("src/test/resources/sample.PPs.vcf", "--pp-as-pl"))
```

Comment:
A couple changes grouped into one, fair point.  Better to just get this in rather than take time to split them out.